pkg/pool-stable-phantom/contracts/StablePhantomPool.sol

```diff
@@ -101,13 +101,6 @@ contract StablePhantomPool is IRateProvider, BaseGeneralPool, ProtocolFeeCache {
     event AmpUpdateStarted(uint256 startValue, uint256 endValue, uint256 startTime, uint256 endTime);
     event AmpUpdateStopped(uint256 currentValue);
 
-    // Since this Pool is not joined or exited via the regular onJoinPool and onExitPool hooks, it lacks a way to
-    // continuously pay due protocol fees. Instead, it keeps track of those internally.
-    // Due protocol fees are expressed in BPT, which leads to reduced gas costs when compared to tracking due fees for
-    // each Pool token. This means that some of the BPT deposited in the Vault for the Pool is part of the 'virtual'
-    // supply, as it belongs to the protocol.
-    uint256 private _dueProtocolFeeBptAmount;
```

Comment:
It is true that we can now mint BPT and don't need to use a join to pay fees, but we might still want to keep this variable around. A proper fee payout involves updating total supply and the collector balances (2 sstores). If we keep this varaiable, we have one extra sload (for the variable), plus the overhead of requiring someone to periodically call here to collect fees.

Given that many operations will be joins and exits (causing fee payouts), the most sensible approach seems to be to keep this optimization and add a `collectDueProtocolFees` function (assuming that it is ok to require the manual collection call).